ButterCMS.Tests/Common.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.IO;
+
+namespace ButterCMS.Tests
+{
+    public static class Common
+    {
+        private static IConfiguration Configuration { get; set; }
+        public static ButterCMSClient SetUpButterClient()
+        {
+            var builder = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json");
```

Comment:
Does this mean that everyone needs to have `appsettings.json` in their project? I'm a little confused why this is under `/tests/`? It seems like this fundamentally changes how to instantiate the ButterCMSClient?

Specifically are these instructions no longer correct?  https://github.com/ButterCMS/buttercms-csharp#usage

Reply:
This is just for supplying an API key to instantiate the client for the test project only. The API value used to be stored in the test project's app.config, but Microsoft has moved away from the old XML style of configs and moved toward .json config files. Users will not have to change their existing code and will instantiate the client the same way as before. 

I added Common.cs because this new json configuration is no longer a simple one-line to bring in the configuration value, but rather two "using" statements and the introduction of the IConfiguration interface. It made more sense to abstract this out to it's own method instead of copy-pasting the setup in every test class. Again, this change only affects the test project and users' code will remain the same.